src/timer/CountDownTimer.test.jsx

```diff
@@ -58,8 +58,8 @@ describe('ExamTimerBlock', () => {
 
     expect(screen.getByRole('alert')).toBeInTheDocument();
     expect(screen.getByText(attempt.exam_display_name)).toBeInTheDocument();
-    expect(screen.getAllByRole('button').length).toEqual(2);
-    expect(screen.getByRole('button', { name: 'Show more' })).toBeInTheDocument();
+    expect(screen.getByText('Show more')).toBeInTheDocument();
+    expect(screen.getAllByRole('button').length).toEqual(1);
```

Comment:
These 'show more' / 'show less' `Alert.Link` components went from buttons to a link (which you would think I could use `getByRole('link')` to grab, but I think because they don't have an href, they aren't really a11y links).